docs/pages/develop/user-interface/fonts.mdx

```diff
@@ -43,9 +44,47 @@ For reference, the following table provides what formats work on which platforms
 
 ## Use a custom font
 
+There are two ways to use a custom font in your project: you can either embed the font in your native project or load it at runtime. The first approach is recommended as it is simpler and more reliable. The second approach is useful if you want to load the font in Expo Go / without creating a new native build that includes it, or load it from a remote URL.
+
+### 1. Embed the font in your native project
+
+> Note: this `expo-font` config plugin is only available for SDK 50 and higher. If you are using an older SDK, you can [load the font at runtime](#2-load-the-font-at-runtime) instead. 
+
+Install [`expo-font`](/versions/latest/sdk/font/#installation) package and add the config plugin to your `app.json` file. The plugin will embed the font in your native project.
+
+```json
+{
+  "expo": {
+    "plugins": [
+      [
+        "expo-font",
+        {
+          "fonts": [
+            "./assets/fonts/Inter-Black.otf"
+          ]
+        }
+      ]
+    ]
+  }
+}
+```
+
+[Create a new native build](/develop/development-builds/create-a-build/), and you can now use the font in your project with the `fontFamily` style prop. The font family name will be the name of the font file, without the extension. For example, if you have a font file named **Inter-Black.otf**, the font family name will be **Inter-Black**.
+
+If you'd like to try out a font without creating a new build, you could alternatively load it at runtime. This is what the second approach explains.
+
+<ConfigReactNative>
+
+- **Android:** Copy font files to **android/app/src/main/assets/fonts**.
+- **iOS**: See ["Adding a Custom Font to Your App" on the Apple Developer documentation](https://developer.apple.com/documentation/uikit/text_display_and_fonts/adding_a_custom_font_to_your_app).
+
+</ConfigReactNative>
+
+### 2. Load the font at runtime
+
 After getting the font file, in your project, you need to install [`expo-font`](/versions/latest/sdk/font/#installation) package.
```

Comment:
```suggestion
After getting the font file, in your project, you need to install [`expo-font`](/versions/latest/sdk/font/#installation) library.
```

Outside the scope of this PR but let's also use library instead of package here. If you think introducing this change here is not worth it, I am happy to audit this doc in a follow up PR.